SnapshotTesting.podspec

```
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = "SnapshotTesting"
-  s.version = "1.6.0"
+  s.version = "1.6.1"
```

```suggestion
  s.version = "1.6.0"
```

Let's keep the version the same for now.